cumulus_library/cli.py

```diff
@@ -113,73 +113,126 @@ def export_all(self, study_dict: Dict):
-        sys.exit("Neither build nor export specified - exiting with no action")
-    study_dict = get_study_dict()
+        study_dict = get_study_dict(args["study_dir"])
+        # here we invoke the cursor once to confirm valid connections
+        builder.cursor.execute("show tables")
 
-    if args["build"]:
-        if "all" in args["target"]:
-            builder.clean_and_build_all(study_dict)
-        else:
-            for target in args["target"]:
-                if target in study_dict:
-                    builder.clean_and_build_study(study_dict[target])
+        if args["build"]:
+            if "all" in args["target"]:
+                builder.clean_and_build_all(study_dict)
+            else:
+                for target in args["target"]:
+                    if target in study_dict:
+                        builder.clean_and_build_study(study_dict[target])
 
-    if args["export"]:
-        if "all" in args["target"]:
-            builder.export_all(study_dict)
-        else:
-            for target in args["target"]:
-                builder.export_study(study_dict[target])
+        if args["export"]:
+            if "all" in args["target"]:
+                builder.export_all(study_dict)
+            else:
+                for target in args["target"]:
+                    builder.export_study(study_dict[target])
 
-    # returning the builder for ease of unit testing
-    return builder
+        # returning the builder for ease of unit testing
+        return builder
 
 
 def get_parser():
     """Provides parser for handling CLI arguments"""
     parser = argparse.ArgumentParser(
-        description="""Generates study views from post-Cumulus ETL data.
-
-        By default, make will remove, and recreate, all table views, assuming
-        you have set the required  credentials, which are used in the following
-        order of preference:
-        - explict command line arguments
-        - cumulus environment variables (CUMULUS_LIBRARY_PROFILE,
-        CUMULUS_LIBRARY_SCHEMA, CUMULUS_LIBRARY_S3, CUMULUS_LIBRARY_REGION)
-        - Normal boto profile order (AWS env vars, ~/.aws/credentials, ~/.aws/config)
-        connecting to AWS. Passing values via the command line will override the
-        environment variables """
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+        description="""Generates study tables and views from post-Cumulus ETL data.
+
+cumulus_library will attempt to create a connection to AWS athena. The
+following order of preference is used to select credentials:
+  - explict command line arguments
+  - cumulus environment variables (CUMULUS_LIBRARY_PROFILE, 
+    CUMULUS_LIBRARY_SCHEMA, CUMULUS_LIBRARY_S3, CUMULUS_LIBRARY_REGION)
+  - Normal boto profile order (AWS env vars, ~/.aws/credentials, ~/.aws/config)""",
+    )
+
+    studygen = parser.add_argument_group("Creating study templates")
+    studygen.add_argument(
+        "-c",
+        "--create",
```

Comment:
nit: Can the user pass multiples of create/build/export? I think you can mark them as an exclusive group. But maybe that would conflict with other groupings. They might make more sense as subcommands, but not an important thought.

Reply:
see above - currently they can.